code/modules/integrated_electronics/subtypes/converters.dm

```diff
@@ -168,6 +202,56 @@
 
 	activate_pin(2)
 
+/obj/item/integrated_circuit/converter/findstring
+	name = "find text"
+	desc = "This gives position of sample in the string. Or returns 0."
+	extended_desc = "first pin is string,to be examined.Second pin is sample,which shoul be found. \
```

Comment:
`The first pin is the string to be examined. The second pin is the sample to be found.`